crypto/x509/x509_vfy.c

```diff
@@ -825,7 +826,7 @@ static int check_trust(X509_STORE_CTX *ctx, int num_untrusted)
     for (i = num_untrusted; i < num; i++) {
         x = sk_X509_value(ctx->chain, i);
         trust = X509_check_trust(x, ctx->param->trust, 0);
-        /* If explicitly trusted return trusted */
+        /* If implicitly or explicitly trusted return trusted */
```

Comment:
The original comment was trying (poorly) to differentiate between "trusted" and "neutral" (not rejected, i.e. merely untrusted). So perhaps just say: "If trusted ..."

Reply:
I see, so I made this more clear as follows:
> If explicitly trusted (so not neutral nor rejected) return trusted